changelog.go

```diff
@@ -0,0 +1,34 @@
+package main
+
+import (
+	"fmt"
+	"strings"
+
+	"gopkg.in/src-d/go-git.v4"
+)
+
+func getLatestChanges(repo string, n int) ([]string, error) {
+	r, err := git.PlainOpen(repo)
+	if err != nil {
+		return nil, fmt.Errorf("failed to open the repo %q: %v", repo, err)
+	}
+	var msgs []string
+	cIter, err := r.Log(&git.LogOptions{})
+	if err != nil {
+		return nil, fmt.Errorf("failed 'git log': %v", err)
+	}
+
+	for i := 0; i < n; i++ {
+		com, err := cIter.Next()
+		if err != nil {
+			return nil, fmt.Errorf("failed to get next commit log: %v", err)
+		}
+
+		if strings.Contains(com.Message, "CHANGELOG") {
```

Comment:
missing colon? it looks like omitting the : on accident would cause panic?

Reply:
I was missing the colon. Added that, but how would it have caused a panic? Contains matches substrings, so I think it still would have worked fine, at least I had no issues manually running it. I will get the test in soon.